pipeline-runner/R/handle_data.R

```diff
@@ -169,7 +169,7 @@ build_qc_response <- function(id, input, error, pipeline_config) {
 send_output_to_api <- function(pipeline_config, input, plot_data_keys, output) {
   c(config, plot_data = plotData) %<-% output
 
-  config <- config[!names(config) %in% c("auth_JWT", "api_url")]
+  config <- config[!names(config) %in% c("auth_JWT", "api_url", "clustering_should_run", "metadataS3Path", "cl_metadata_bucket")]
```

Comment:
maybe add comment as to why we do this. I personally did not know it was to avoid storing in the processing config in S3

Reply:
Added comment